third_party/libunwindstack/PeCoffEpilog.cpp

```diff
@@ -45,11 +45,11 @@ class PeCoffEpilogImpl : public PeCoffEpilog {
   // can be retrieved using GetLastError() and registers 'regs' are not updated.
   bool DetectAndHandleEpilog(uint64_t function_start_address, uint64_t function_end_address,
                              uint64_t current_offset_from_start_of_function, Memory* process_memory,
-                             Regs* regs) override;
+                             Regs* regs, bool* in_epilog) override;
```

Comment:
Nit: Sometimes pointer names indicate whether they are "in" or "out" parameters by having one of them or both in the name such as "in_out_value". This could be read as a pointer value that is the input. That doesn't make sense here so feel free to disregard. Maybe "is_in_epilog" is a little bit clearer.